sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -722,6 +722,7 @@
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=programYearTotalFullTimeSBSD + programYearTotalPartTimeSBSD" target="programYearTotalSBSD" />
+          <zeebe:output source="=programYearTotalFullTimeCSGP + programYearTotalPartTimeCSGP" target="programYearTotalCSGP" />
```

Comment:
The variable declarations seem to be in a different position from the other files. Aren't they exact copies?

Reply:
Yeah it should be, I believe the element which was present is different in the dmn's differently.

Reply:
I am not sure if we are talking about the same point. Let's sync once possible on this one.